Makefile

```diff
@@ -116,12 +118,24 @@ pending-tests:
 
 pt: pending-tests
 
+clj-kondo-lint:
+	clj-kondo --lint src:test:build.clj
+
+clj-kondo-lint-ci:
+	clj-kondo --lint src:test:build.clj --config .clj-kondo/ci-config.edn
+
+cljfmt-check:
+	cljfmt check src dev test build.clj
+
+cljfmt-fix:
+	cljfmt fix src dev test build.clj
+
 test: cljtest cljstest nodejs-test browser-test
 
 eastwood:
 	clojure -M:dev:cljtest:eastwood
 
-ci: test eastwood
+ci: test eastwood clj-kondo-lint-ci cljfmt-check
```

Comment:
We should add `cljfmt` and `clj-kondo` to the Prerequisites section of the readme so others will be able to contribute.